applications/GeoMechanicsApplication/tests/point_load_semi_analytical_solution.py

```diff
@@ -0,0 +1,230 @@
+                ta = tr + eps1
+            tt = ta * ta
+            xr = math.asin(math.sqrt((tr * tr - nn) / (tt - nn)))
+            xa = 0.0
+            xb = xr - eps1
+            k = int(1000.0 * (xb - xa))
+            dx = (xb - xa) / k
+            fa = 0.0
+            g = 0.0
+            for j in range(k+1):
+                s = math.sin(xa + j * dx)
+                ss = s * s
+                yy = nn + (tt - nn) * ss
+                a = (1.0 - 2.0 * yy) * (1.0 - 2.0 * yy) * (tt - nn) * ss
+                b = (1.0 + 8.0 * yy * (-1.0 + yy * (3.0 - 2.0 * nn - 2.0 * yy * (1.0 - nn))))
+                f = a / b
+                if j > 0:
+                    g -= fac * (f + fa) * dx
+                fa = f
+
+            xa = xr + eps1
+            xb = pi / 2.0
+            k = int(1000.0 * (xb - xa))
+            dx = (xb - xa) / k
+            for j in range(k+1):
+                s = math.sin(xa + j * dx)
+                ss = s * s
+                yy = nn + (tt - nn) * ss
+                a = (1.0 - 2.0 * yy) * (1.0 - 2.0 * yy) * (tt - nn) * ss
+                b = (1.0 + 8.0 * yy * (-1.0 + yy * (3.0 - 2.0 * nn - 2.0 * yy * (1.0 - nn))))
+                f = a / b
+                if j > 0: 
+                    g -= fac * (f + fa) * dx
+                fa = f
+
+            xr = math.asin(math.sqrt((tr * tr - 1.0) / (tt - 1.0)))
+            xa = 0.0
+            xb = xr - eps1
+            k = int(1000.0 * (xb - xa))
+            dx = (xb - xa) / k
+            for j in range(k+1):
+                s = math.sin(xa + j * dx)
+                ss = s * s
+                yy = 1.0 + (tt - 1.0) * ss
+                a = 4.0 * yy * (tt - 1.0) * (yy - nn) * ss
+                b = (1.0 + 8.0 * yy * (-1.0 + yy * (3.0 - 2.0 * nn - 2.0 * yy * (1.0 - nn))))
+                f = a / b
+                if j > 0:
+                    g -= fac * (f + fa) * dx
+                fa = f
+
+            xa = xr + eps1
+            xb = pi / 2.0
+            k = int(1000.0 * (xb - xa))
+            dx = (xb - xa) / k
+            for j in range(k+1):
+                s = math.sin(xa + j * dx)
+                ss = s * s
+                yy = 1.0 + (tt - 1.0) * ss
+                a = 4.0 * yy * (tt - 1.0) * (yy - nn) * ss
+                b = (1.0 + 8.0 * yy * (-1.0 + yy * (3.0 - 2.0 * nn - 2.0 * yy * (1.0 - nn))))
+                f = a / b
+                if j > 0:
+                    g -= fac * (f + fa) * dx
+                fa = f
+        else:
+            ta = t
+            if t > (tr - eps):
+                ta = tr - eps1
+            tt = ta * ta
+            xa = 0.0
+            xb = pi / 2.0
+            k = int(1000.0 * (xb - xa))
+            dx = (xb - xa) / k
+            g = 0.0
+            for j in range(k+1):
+                s = math.sin(xa + j * dx)
+                ss = s * s
+                yy = nn + (tt - nn) * ss
+                a = (1.0 - 2.0 * yy) * (1.0 - 2. * yy) * (tt - nn) * ss
```

Comment:
Nitpicking (for consistency):
```suggestion
                a = (1.0 - 2.0 * yy) * (1.0 - 2.0 * yy) * (tt - nn) * ss
```